quinn-proto/src/config.rs

```diff
@@ -444,10 +445,10 @@ where
     }
 
     /// Private key used to authenticate data included in handshake tokens.
-    pub fn token_key(&mut self, value: &[u8]) -> Result<&mut Self, ConfigError> {
-        self.token_key = Arc::new(S::HmacKey::new(value)?);
-        Ok(self)
-    }
+    // pub fn token_key(&mut self, value: &[u8]) -> Result<&mut Self, ConfigError> {
+    //     self.token_key = Arc::new(S::HmacKey::new(value)?);
+    //     Ok(self)
+    // }
```

Comment:
Is this API used anywhere? The test didn't fail when I commented it out.
Should I update it or is it dead code?

Reply:
It's part of the public interface and should be retained. Downstream code might want to specify it to allow servers behind a load balancer to statelessly accept tickets issued by eachother, or to allow a server to accept its own tokens after a restart.